lib/_http_outgoing.js

```diff
@@ -458,8 +451,8 @@ function processHeader(self, state, key, value, validate) {
     validateHeaderName(key);
   if (ArrayIsArray(value)) {
     if (value.length < 2 || !isCookieField(key)) {
-      for (var i = 0; i < value.length; i++)
-        storeHeader(self, state, key, value[i], validate);
+      for (const valueItem of value)
```

Comment:
We should not make this change for performance reasons.

Reply:
resolved in https://github.com/nodejs/node/pull/30958/commits/17049d56621869d8776a922d5e56b6c5ec35be7b